src/console-errors/wiiu/support-codes/codes/101-fp.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 	'0109': new SupportCode('CORE_INVALID_INDEX', 'Missing description', 'Missing message', 'Missing fix', 'Missing link'),
 	'0110': new SupportCode('CORE_OUT_OF_MEMORY', 'Missing description', 'Missing message', 'Missing fix', 'Missing link'),
 	'0111': new SupportCode('CORE_INVALID_ARGUMENT', 'Missing description', 'Missing message', 'Missing fix', 'Missing link'),
-	'0112': new SupportCode('CORE_TIMEOUT', 'Missing description', 'Unable to connect to the server.\n\nPlease try again later.\n\nIf the problem persists, please\nmake a note of the error code and\nvisit support.nintendo.com.', 'Missing fix', 'Missing link'),
+	'0112': new SupportCode('CORE_TIMEOUT', 'Connection timeout', 'Unable to connect to the server.\nPlease try again later', 'Verify your wifi is working. If it is\nworking, then this is a serverside error.\nPlease wait and try again later.', 'Missing link'),
```

Comment:
Why was the message changed here? The message should be as it appears on the console. These were extracted from the consoles MSBT files, so they should already be how the error displays on the console?

Reply:
WiiU description was changed in error, sorry about that. It's fixed now!